includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -893,6 +893,25 @@ public function link_session_to_customer( $session_id, $customer_id ) {
 		);
 	}
 
+	/**
+	 * Sends the contents of the "forterToken" cookie to the server.
+	 *
+	 * @param string $token Contents of the "forterToken" cookie, used to identify the current browsing session.
+	 *
+	 * @return array An array, containing a `success` flag.
+	 *
+	 * @throws API_Exception If an error occurs.
+	 */
+	public function send_forter_token( $token ) {
+		return $this->request(
+			[
+				'token'  => $token,
```

Comment:
Should we send here the `userAgent` and the merchant `IP` too? Or should we assume Forter will track that with the JS?

Reply:
I think we could send it here too, just to be safe. But maybe we should check with them before committing to one approach.

Reply:
Also, there's an extra space after `'token'`.